packages/ui/src/components/molecules/ProductCard/styles.scss

```diff
@@ -174,6 +184,32 @@
     height: fit-content;
   }
 
+  [data-fs-product-card-delivery-badges] {
+    display: flex;
+    flex-direction: column;
+    row-gap: var(--fs-spacing-0);
+    font-size: var(--fs-product-card-delivery-badge-text-size);
+
+    [data-fs-product-card-delivery-badge] {
+      &::before {
+        display: inline-block;
+        width: var(--fs-product-card-delivery-badge-status-width);
+        height: var(--fs-product-card-delivery-badge-status-width);
+        margin-right: var(--fs-spacing-1);
+        content: "";
+        border-radius: var(--fs-product-card-delivery-badge-status-border-radius);
+      }
+
+      &[data-fs-product-card-delivery-badge-availability="true"]::before {
+        background-color: var(--fs-product-card-delivery-badge-status-available);
```

Comment:
@renatamottam we don't have this feedback color mentioned in the figma in our tokens. So I used the `--fs-color-success-bkg`. Do you think we should add it?

<img width="796" height="132" alt="image" src="https://github.com/user-attachments/assets/915ae064-857e-4df1-98ae-edba85ffbbd3" />

I think the `--fs-color-success-bkg` is too light for this context :/ 
<img width="296" height="448" alt="image" src="https://github.com/user-attachments/assets/9227cdf4-6626-4fdd-9b97-77704f758c15" />